public/react-services/pattern-handler.js

```diff
@@ -22,18 +22,19 @@ export class PatternHandler {
    */
   static async getPatternList(where) {
     try {
-      var patternList = await SavedObject.getListOfWazuhValidIndexPatterns();
+      let patternList = await SavedObject.getListOfWazuhValidIndexPatterns(AppState.getCurrentPattern(), where); // check (AppState.getCurrentPattern() is not an array)
```

Comment:
It should send as param the current pattern selected and the default pattern from wazuh.yml
may be
```
const defaultPatterns = [AppState.getCurrentPattern(), config.pattern]];
let patternList = await SavedObject.getListOfWazuhValidIndexPatterns(defaultPatterns, where);
```